ml_packages_in_R.rmd

```diff
@@ -0,0 +1,123 @@
+#Low-key ML packages in R
+#Author - Vedant Kumar and Siddhant Kumar
+
+#Part 1 - Nuralnet Package in R
+
+```{r}
+# install package
+install.packages("neuralnet")
```

Comment:
Do not evaluate any install.packages() statements in your document. Consumers of an Rmd file won’t want packages to get installed when they knit your document.